### ISSUE TITLE
Disable the linter in the generated code

### ISSUE THREAD

**Issue by alexgarzao** [opencodeco/validgen#73]
In some cases, the linter generates numerous warnings when scanning the generated code.

Below is an example:

```
if !(obj.boolField == true) { 
```

In this task, the idea is to add a linter directive to ignore the generated code:

```
//lint:ignore S1002 intentional explicit comparison
if !(obj.boolField == true) { 
```

Another possibility is to [add a directive](https://golangci-lint.run/docs/linters/false-positives/#nolint-directive) that forces the linter to ignore all the files generated by ValidGen.

```
// Code generated by ValidGen. DO NOT EDIT.

//nolint:unparam
package main
...
```
